docs/pages/example/snapshot.js

```diff
@@ -0,0 +1,10 @@
+/*---
+title: Snapshot
```

Comment:
In line with the rest of the examples, I think we should use something like "Snapshot an image of a map" as the title, "snapshot" on it's own is a bit unclear.

Reply:
I would make the case that the description 'capture a snapshot image of a map' and the interactive example 'click on the map' make this clear. Shorter titles also make it easier and faster to scan through the example titles..

However, I understand that this is your project (styles, decisions, aesthetics, etc.).

![screenshot from 2018-04-25 10-03-02](https://user-images.githubusercontent.com/16657983/39236902-edb2151a-4871-11e8-84c3-2f33ae0ebfde.png)

Reply:
I'm just working off the instructions at https://github.com/mapbox/mapbox-gl-js/tree/master/docs#writing-examples which say: "A short title for the example in sentence case as a verb phrase".

Reply:
Title updated as suggested.